### PR TITLE
Test/build formula

### PR DESCRIPTION
**Pull request by DittrichLucas**
**- What I did**
- Add build formula test
- Separate tests in suites
- Add job for build formula test
- Separate jobs for running tests

**- How I did it**
- Moving and adding files
- Adding a new job to the pipeline

**- How to verify it**
- Follow pipeline
- Run in the terminal: `go test -v ./.circleci/validation/structure/.` and `go test -v ./.circleci/validation/make/.`

**- Description for the changelog**
Add build formula test, separate tests in suites and jobs for run tests

### CODE REVIEW

File: .circleci/validation/make/makebuild_test.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package make
+
+import (
+	"fmt"
+	"testing"
+)
+
+var root = "../../../"
```
Comment (victor-schumacher) [inline .circleci/validation/make/makebuild_test.go:8]: It can be a const?

File: .circleci/validation/structure/utilities.go [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
 )
 
+func check(err error) {
+	if err != nil {
+		fmt.Printf("Erro: %v\n", err)
```
Comment (victor-schumacher) [inline .circleci/validation/structure/utilities.go:12]: Should be Error instead of Erro?

File: .circleci/validation/make/utilities.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+func returnFilesFromFolder(root string) ([]string, error) {
+	var files []string
+	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		files = append(files, path)
+		return nil
+	})
+	return files, err
+}
+
+func validateMakefile(path string) (string, bool) {
```
Comment (kaduartur) [inline .circleci/validation/make/utilities.go:51]: could you return just error?

File: .circleci/validation/make/utilities.go [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+			path = strings.ReplaceAll(path, "/", " ")
+			success = fmt.Sprintf("Build success: %s", path)
+		}
+	}
+
+	return success, false
+}
+
+func runBuild(path string) (bool, error) {
```
Comment (kaduartur) [inline .circleci/validation/make/utilities.go:75]: could you return just error?

**Pull request closed without merging**